k8s/wordpress/README.md

```diff
@@ -191,7 +208,35 @@ an external IP address for your WordPress app, run the following command:
 ```
 kubectl patch svc "$APP_INSTANCE_NAME-wordpress-svc" \
   --namespace "$NAMESPACE" \
-  --patch '{"spec": {"type": "LoadBalancer"}}'
+  --patch '{"spec": {"type": "NodePort"}}'
+
+APPLICATION_UID=$(kubectl get applications/${APP_INSTANCE_NAME} --namespace="$NAMESPACE" --output=jsonpath='{.metadata.uid}')
+
+cat <<EOF | kubectl create --namespace "$NAMESPACE" -f -
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: $APP_INSTANCE_NAME-wordpress-ingress
+  labels:
+    app.kubernetes.io/name: $APP_INSTANCE_NAME
+    app.kubernetes.io/component: wordpress-webserver
+  ownerReferences:
+  - apiVersion: app.k8s.io/v1beta1
+    blockOwnerDeletion: true
+    controller: true
+    kind: Application
+    name: $APP_INSTANCE_NAME
+    uid: $APPLICATION_UID
+spec:
+  tls:
+  - secretName: $APP_INSTANCE_NAME-tls
+  backend:
+    serviceName: $APP_INSTANCE_NAME-wordpress-svc
+    servicePort: http
+EOF
+
+kubectl patch application "$APP_INSTANCE_NAME" --type=json \
+  -p='[{"op": "add", "path": "/spec/componentKinds/-", "value":{ "group": "extensions/v1beta1", "kind": "Ingress" } }]'
 ```
 
```

Comment:
In case of manual installation via CLI, we could treat it as optional, but we could add an owner reference for the tls secret as well.